packages/tasit-contracts/decentraland/scripts/install.sh

```diff
@@ -8,17 +8,43 @@ then
 fi
 
 DECENTRALAND_DIR="$PROJECT_DIR/decentraland";
-REPOS="mana land marketplace-contracts";
 
-for repo in $REPOS;
-do
-    REPO_DIR="$DECENTRALAND_DIR/$repo";
+# mana
+# Note: mana repository has no tags and there is no branch that can be used.
+REPO="mana"
 
-    if [ ! -e "$REPO_DIR/package.json" ];
-    then
-        rm -rf $REPO_DIR;
-        git clone https://github.com/decentraland/$repo.git $REPO_DIR;
-    fi
+REPO_DIR="$DECENTRALAND_DIR/$REPO";
 
+if [ ! -e "$REPO_DIR/package.json" ];
+then
+    rm -rf $REPO_DIR;
+    git clone https://github.com/decentraland/$REPO.git $REPO_DIR;
+    npm i --prefix $REPO_DIR;
+fi
+
+
+# land
+REPO="land"
+TAG="deploy/2018-08-31"
```

Comment:
I know I said tag before, but I'm wondering if the hash of the commit itself would be better, since tags can be changed to point to different commits

Reply:
Either way this is a big improvement already

Reply:
OK, I'll commit this change in a few minutes.

Reply:
I'm resolving this